backend/app/services/llm_dispatch.py

```
@@ -0,0 +1,41 @@
+        "request_id": payload.request_id,
+        "max_cost_cents": payload.max_cost_cents,
+    }
+
+
+def enqueue_llm_task(task_name: str, payload: LLMTaskPayload):
+    """
+    Enqueue a deterministic LLM task using the canonical payload contract.
+    """
+    task = _TASK_MAP.get(task_name)
+    if task is None:
+        raise ValueError(f"Unknown LLM task name: {task_name}")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)